storage/oasis/nodeapi/cobalt/convert.go

```diff
@@ -153,18 +153,24 @@ func ConvertGenesis(d genesisCobalt.Document) *genesis.Document {
 		runtimes[i] = convertRuntime(r)
 	}
 
-	return &genesis.Document{
-		Height:  d.Height,
-		Time:    d.Time,
-		ChainID: d.ChainID,
+	suspendedRuntimes := make([]*registry.Runtime, len(d.Registry.SuspendedRuntimes))
+	for i, r := range d.Registry.SuspendedRuntimes {
+		suspendedRuntimes[i] = convertRuntime(r)
+	}
+
+	return &nodeapi.GenesisDocument{
+		Height:    d.Height,
+		Time:      d.Time,
+		ChainID:   d.ChainID,
+		BaseEpoch: uint64(d.Beacon.Base),
 		Governance: governance.Genesis{
 			Proposals:   proposals,
 			VoteEntries: voteEntries,
 		},
 		Registry: registry.Genesis{
 			Entities:          d.Registry.Entities,
-			Runtimes:          []*registry.Runtime{},
```

Comment:
Yeah, luckily, I noticed this since I used the cobalt/convert version as a baseline for the added eden/convert. And since we have e2e regression tests for eden, it caught this error.